website/docs/about/Goals.md

```diff
@@ -11,9 +11,11 @@ import Disclaimer from '../\_disclaimer.mdx';
 
 The primary goal of UNTP is to make significant reductions in the incidence of greenwashing by giving unsustainable behaviour nowhere to hide. This will also uplift the value of legitimate ESG credentials from supply chain actors that have implemented sustainable practices. UNTP will have achieved it's purpose when
 
-- Most supply chain shipments are accompanied by verifiable ESG performance data.
-- Greenwashing is a niche activity that is easily detected and quickly penalised by markets and regulators.
-- Products with the best sustainability characteristics enjoy the greatest market access and price uplift.
+| Goals                         | Evidence and examples |
+| ----------------------------- | ----------- |
+| Most supply chain shipments are accompanied by verifiable ESG performance data. | In complex supply chains this means that each step and at every depth high quality verifiable product information accompanies products via a Digital Product Passport |
```

Comment:
'Depth' is not self-explanatory, maybe `...that each step and [reviewing any number of previous steps] high quality...'?